ush/parsing_namelists_FV3.sh

```diff
@@ -323,17 +315,12 @@ cat >> input.nml <<EOF
   do_sppt      = ${do_sppt:-".false."}
   do_shum      = ${do_shum:-".false."}
   do_skeb      = ${do_skeb:-".false."}
-EOF
-
-if [ $cpl = .true. ]; then
-  cat >> input.nml << EOF
   frac_grid    = ${FRAC_GRID:-".true."}
   cplchm       = ${cplchem:-".false."}
   cplflx       = $cplflx
   cplice       = ${cplice} 
```

Comment:
Are these last three guaranteed to be set even if cpl is ".false."? If not, they should have defaults of ".false.".

Reply:
It looked to be like they are set: https://github.com/NOAA-EMC/global-workflow/blob/develop/parm/config/config.fcst#L47-L51 but happy to add the false if that's what is wanted.

Reply:
Made the update @WalterKolczynski-NOAA  -- always better to have a default after all.